### PR TITLE
Vj urban command

### PR DESCRIPTION
**Pull request by TitanVJ**
added embed util function to helper_files
added settings.py. it'll contain things that cannot be initialized when a cog loads for watever reason. 

in main.py's on_ready added lines to init the vars in settings.py

### CODE REVIEW

File: helper_files/settings.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# TODO: Come up with better name for this file
+# Use this hold globals and other things need across the cogs
+# that cannot be initialized for whatever reason in the cogs themselves
+
+# To be initialed from main in the on_ready function
+# Used in almost all the cogs for author name and author avatar in embeds
+global BOT_NAME
```
Comment (modernNeo) [inline helper_files/settings.py]: @TitanVJ  why have a separate file for these instead of just putting it in `main.py`?

Comment (TitanVJ) [reply helper_files/settings.py]: i dont like the idea of importing main. also file could be useful for storing other things of similar nature in the future, would prevent main from getting cluttered

Comment (modernNeo) [reply helper_files/settings.py]: @TitanVJ is the intention of the file that it will store all variables that get initialized/used in main.py?

Comment (TitanVJ) [reply helper_files/settings.py]: yeah i guess, but also for anything thats used in a cog that cant be initialized in the cog.

Comment (modernNeo) [reply helper_files/settings.py]: ok.

**Comment by TitanVJ** [comment]
gonna re make pr from csss's wall_e repo

**Pull request closed without merging**